src/config/clankerTokenV4.ts

```diff
@@ -0,0 +1,385 @@
+          DEFAULT_SUPPLY,
+          cfg.tokenAdmin,
+          cfg.image,
+          metadata,
+          socialContext,
+          BigInt(cfg.chainId),
+        ],
+        cfg.tokenAdmin,
+        '0x4b07',
+        { chainId: cfg.chainId }
+      )
+    : {
+        salt: '0x0000000000000000000000000000000000000000000000000000000000000000',
+        token: undefined,
+      };
+
+  return {
+    address: CLANKER_FACTORY_V4,
+    abi: Clanker_v4_abi,
+    functionName: 'deployToken',
+    args: [
+      {
+        tokenConfig: {
+          tokenAdmin: cfg.tokenAdmin,
+          name: cfg.name,
+          symbol: cfg.symbol,
+          salt: salt as `0x${string}`,
+          image: cfg.image,
+          metadata,
+          context: socialContext,
+          originatingChainId: BigInt(cfg.chainId),
+        },
+        lockerConfig: {
+          locker: CLANKER_LOCKER_V4,
+          rewardAdmins: cfg.rewards.recipients.map(({ admin }) => admin),
+          rewardRecipients: cfg.rewards.recipients.map(({ recipient }) => recipient),
+          rewardBps: cfg.rewards.recipients.map(({ bps }) => bps),
+          tickLower: cfg.pool.positions.map(({ tickLower }) => tickLower),
+          tickUpper: cfg.pool.positions.map(({ tickUpper }) => tickUpper),
+          positionBps: cfg.pool.positions.map(({ positionBps }) => positionBps),
+          lockerData: '0x',
+        },
+        poolConfig: {
+          pairedToken: cfg.pool.pairedToken,
+          tickIfToken0IsClanker: cfg.pool.tickIfToken0IsClanker,
+          tickSpacing: cfg.pool.tickSpacing,
+          hook: hook,
+          poolData: poolData,
+        },
+        mevModuleConfig: {
+          mevModule: CLANKER_MEV_MODULE_V4,
+          mevModuleData: '0x',
+        },
+        extensionConfigs: [
+          // vaulting extension
+          ...(cfg.vault
+            ? [
+                {
+                  extension: CLANKER_VAULT_V4,
+                  msgValue: 0n,
+                  extensionBps: cfg.vault.percentage * 100,
+                  extensionData: encodeAbiParameters(VAULT_EXTENSION_PARAMETERS, [
+                    cfg.tokenAdmin,
+                    BigInt(cfg.vault.lockupDuration),
+                    BigInt(cfg.vault.vestingDuration),
+                  ]),
+                },
+              ]
+            : []),
+          // airdrop extension
+          ...(cfg.airdrop
+            ? [
+                {
+                  extension: CLANKER_AIRDROP_V4,
+                  msgValue: 0n,
+                  extensionBps: cfg.airdrop.percentage * 100,
+                  extensionData: encodeAbiParameters(AIRDROP_EXTENSION_PARAMETERS, [
+                    cfg.airdrop.merkleRoot,
+                    BigInt(cfg.airdrop.lockupDuration),
+                    BigInt(cfg.airdrop.vestingDuration),
+                  ]),
+                },
+              ]
+            : []),
+          // devBuy extension
+          ...(cfg.devBuy && cfg.devBuy.ethAmount !== 0
```

Comment:
People can technically do multiple dev buys on the same deployment, Idk why we'd enable that, but it is a thing that doesn't map cleanly here